src/app/public/modules/colorpicker/types/colorpicker-result.ts

```diff
@@ -1,4 +1,10 @@
 import { SkyColorpickerOutput } from './colorpicker-output';
- export interface SkyColorpickerResult {
+/**
+ * Specifies the selected color.
```

Comment:
This type is used when the users **apply** a color. Should we change this to say something to match the description you used in the `SkyColorpickerComponent` ?

Reply:
How about `Indicates the color that users apply when they select Apply in the colorpicker.`

Reply:
Works for me!